python/sbp/client/examples/pusher_client.py

```diff
@@ -65,13 +68,12 @@ def connect_it(data):
       push_client.connection.bind('pusher:connection_established', connect_it)
       push_client.connect()
 
-      handler.add_callback(push_it, [SBP_MSG_OBS, SBP_MSG_BASE_POS, SBP_MSG_OBS_DEP_A])
-
       try:
-        while True:
-          time.sleep(0.1)
+        for delta, time, msg in handler.filter(OBS_MSG_LIST):
```

Comment:
I'm going to change the convention to be a 2-tuple of the message and a dictionary of metadata that may contain delta and timestamp if these are available, as well as other fields that we may add in future.  I'll also change the call signature to take the metadata as **kwargs.